src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball " + "with "
+                + color.toLowerCase() + " color "
+                + "and number " + number;
+    }

```

Comment:
The toString method implementation is calling toLowerCase() on the color field. While this is not incorrect, it's worth considering if the color should always be represented in lower case for all usages. If the color should maintain the case it was set with, you might want to remove the toLowerCase() call. If lower case is a requirement for the color representation, consider enforcing this at the time the color is set, within the setColor method.